app/src/main/java/com/example/footprnt/Map/Util/MapUtil.java

```diff
@@ -179,4 +197,149 @@ public static JSONObject getContinents(Activity activity){
         return mContinents;
     }
 
+    /**
+     * On search result, locates place on map
+     *
+     * @param searchText
+     * @param map
+     * @param context
+     */
+    public static void geoLocate(EditText searchText, GoogleMap map, Context context){
+        String searchString = searchText.getText().toString();
```

Comment:
Just double checking, but if the text box is empty does searchText.getText() return an empty String or null?